modules/ldap/src/main/java/com/nhds/wasabi/ldap/CachedUserDirectory.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * Wasabi-LDAP Copyright 2018 Neighborhoods.com
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.nhds.wasabi.ldap;
```

Comment:
Thinking about our conversation more, I think you are right that this should be `com.neighborhoods`. Adding this comment to remind myself to do so (at the same time that I receive/implement your feedback).